README.md

```diff
@@ -107,6 +107,10 @@ Supported crates such as `cargo-binstall` itself can also be updated with `cargo
   - You can find a full description of errors including exit codes here: <https://docs.rs/binstalk/latest/binstalk/errors/enum.BinstallError.html>
 - Can I use it in CI?
   - Yes! For GitHub Actions, we recommend the excellent [taiki-e/install-action](https://github.com/marketplace/actions/install-development-tools), which has explicit support for selected tools and uses `cargo-binstall` for everything else.
+  - Additionally, we also provide a minimalistic GitHub Action that installs `cargo-binstall`:
+    ```yml
+      - uses: cargo-bins/cargo-binstall@main
```

Comment:
Having the ability to install in GitHub Action by specifying our repository url sounds great!

Although I'm also a little bit worry about mixing it up, but it does sound really convenient and straight-forward.

cc @passcod What's your opinion on this PR?

Reply:
Another popular pattern is having either separate:
- repository, e.g.: `- uses: cargo-bins/setup-cargo-binstall@v1` (makes versioning very easy),
- or (less popular) branch or tag, e.g.: `- uses: cargo-bins/cargo-binstall@gha-setup`

Reply:
Given that we also have the `install-from-binstall-release.sh` within the same repository, it does make sense to also make it an action since we just need to add a simple `action.yml`.

Reply:
Yeah, that action is surprisingly simple! I'd be glad to have it. However it would be nice to have an integration test in this PR as well, so we don't accidentally break the action by breaking the scripts.

Reply:
I made separate yml file to test it: .github/workflows/gh-action.yml, but let me know if you'd prefer to have it as part of .github/workflows/install-script.yml